compute_endpoint/globus_compute_endpoint/engines/globus_compute.py

```diff
@@ -86,10 +90,15 @@ def __init__(
             executor = HighThroughputExecutor(  # type: ignore
                 *args,
                 label=label,
+                encrypted=encrypted,
                 **kwargs,
             )
         self.executor = executor
 
+    @property
+    def encrypted(self):
+        return self.executor.encrypted
+
```

Comment:
There's not much to test here in this PR, but consider adding a UT for this property.